### PR TITLE
Documentation: Adjust styling and add Appetize link in the modal

### PR DESCRIPTION
**Pull request by jozan**
- Hover effect for simulator image
- "Run this example" is now a link and opens the modal
- Added Appetize.io link in the modal
- Adjusted simulator iframe size

### CODE REVIEW

File: website/src/react-native/css/react-native.css [hunk 1]
```diff
@@ -890,22 +890,24 @@ small code, li code, p code {
...
-.modal-content iframe {
-  margin: 0 auto;
-}
+.modal-content iframe { margin: 0 auto; }
 
 .modal-button-open {
+  cursor: pointer;
   text-align: center;
 }
+  .modal-button-open-img {
```
Comment (vjeux) [inline website/src/react-native/css/react-native.css]: why do you indent this?

File: website/layout/AutodocsLayout.js [hunk 2]
```diff
@@ -492,7 +492,8 @@ var Modal = React.createClass({
           <div className="modal-content">
             <button className="modal-button-close">&times;</button>
             <div className="center">
-              <iframe className="simulator" src={url} width="274" height="550" frameborder="0" scrolling="no"></iframe>
+              <iframe className="simulator" src={url} width="256" height="550" frameborder="0" scrolling="no"></iframe>
+              <p>Powered by <a href="https://appetize.io">appetize.io</a></p>
```
Comment (vjeux) [inline website/layout/AutodocsLayout.js]: target="_blank"

File: website/src/react-native/css/react-native.css (hunk 1 above)
Comment (jozan) [inline website/src/react-native/css/react-native.css]: To represent hierarchy in CSS, just a habit.

Comment (jozan) [inline website/src/react-native/css/react-native.css]: I'll remove indentation

File: website/layout/AutodocsLayout.js (hunk 2 above)
Comment (jozan) [inline website/layout/AutodocsLayout.js]: Ahh, right. I didn't check how it was done in the showcase page.

**Pull request merged**